rollingpin/transports/ssh.py

```diff
@@ -64,8 +64,7 @@ def getPublicKey(self):
             return succeed(public_key)
 
     def getPrivateKey(self):
-        key = self.factory.key.keyObject
-        return succeed(key)
+        return succeed(self.factory.key)
```

Comment:
🔕 Does this work in old twisted as well? Perhaps all of your commits should be squashed into one.